WooCommerce/Classes/Extensions/UITableView+Helpers.swift

```diff
@@ -0,0 +1,19 @@
+import UIKit
+
+extension UITableView {
+
+    /// Return the last Index Path (the last row of the last section) if available
+    func lastIndexPath() -> IndexPath? {
+        guard numberOfSections > 0 else {
+            return nil
+        }
+        let section = numberOfSections - 1
+
+        guard numberOfRows(inSection: section) > 0 else {
+            return nil
```

Comment:
nit: I just thought of an edge case where the last section has no rows but an earlier section does. `lastIndexPath` sounds like it would trace back to the last index path that exists in the table view. if this is not case, maybe a name like `lastIndexPathOfTheLastSection` would be more clear that it's only checking the last section?

Reply:
`lastIndexPathOfTheLastSection` sounds better to me 👍